.github/workflows/next-scan.yml

```diff
@@ -42,4 +47,28 @@ jobs:
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         SONAR_TOKEN: ${{ fromJSON(steps.secrets.outputs.vault).sq_next_token }}
-        SONAR_HOST_URL: https://next.sonarqube.com/sonarqube/
\ No newline at end of file
+        SONAR_HOST_URL: https://next.sonarqube.com/sonarqube/
+    - name: SonarQube Cloud EU Scan
```

Comment:
Nitpick: since we added these two new scans, perhaps we should rename the job from "next" to something like "code-analysis"?

Reply:
Makes a lot of sense, what about sonarqube? 
Since all of them are variations of SQS and SQC?

Reply:
<img width="727" height="262" alt="image" src="https://github.com/user-attachments/assets/4c11e2f5-a126-4a0c-bb3c-cee8559208df" />